Update/omot_005.txt

```diff
@@ -1156,7 +1156,7 @@ void main()
 //rそう言って鷹野は、ぽむぽむ、と沙都子の頭を優しく撫でる。沙都子も『むぅ…』と少し拗ねたようにむずがりながら、気持ちがいいのかなすがままに任せていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "そう言って鷹野は、ぽむぽむ、と沙都子の頭を優しく撫でる。沙都子も『むぅ…』と少し拗ねたようにむずがりながら、気持ちがいいのかなすがままに任せていた。",
-			NULL, "Takano gently stroked Satoko's head after saying that. Satoko uttered a faintly sulking \"Muu...\" but didn't fight it.", Line_Normal);
+			NULL, "Takano gently stroked Satoko's head after saying that. Satoko uttered a faintly sulking \"Muu...\" but didn't fight it thanks to her comforts.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Takano gently stroked Satoko's head after saying that. Satoko uttered a faintly sulking \"Muu...\" but she might've actually liked it because she didn't fight it.", Line_Normal);
```